PANCHANGE_workflow_2020-07-20.txt

```diff
@@ -0,0 +1,17 @@
+PANCHANGE Naming Conventions
+ 
+Branch name: stage_task_initials. e.g. cleaning_OCI_chunk_KT 
+Different stages include cleaning, merging, analyses, sensitivity, visualisation. 
+
+File naming: PANCHANGE_purpose_date. e.g. PANCHANGE_descriptives_2020-07-14.Rmd
+
+Workflow:
+- Log ongoing tasks by creating an Issue on GitHub
+- Make sure to assign another team member to an Issue, so it is clear who is working on which issue.
+- Make sure to attach each issue with the appropriate label. If there is an Issue that you think warrants discussion in a weekly meeting then make sure to add the tag "Meeting topic" and this will be discussed in the next meeting.  
+- Create a new branch when starting a new task and never edit the master script. 
+- Each team member can work simultaneously on the scripts using different branches, however make sure there is no task overlap.
+- All code will be reviewed by another member of the team before pushing to the master script. Therefore, make sure to assign a member of the team on Github to review your push. 
+
```

Comment:
Should maybe add information here about deleting the branches once they are completed? @knthompson26

Reply:
I'll approve this file addition but will add this as an issue to include that detail here :) Otherwise I think this looks good! @knthompson26 

Once you submit the new review with this change, I'd recommend a) using a new branch name so you can review the scripts with this one and b) assigning Kirstin to do the final review so she can say whether there's anything else that needs adding/changing in this workflow.